fvtest/porttest/si.cpp

```diff
@@ -1690,19 +1690,17 @@ TEST(PortSysinfoTest, sysinfo_test_get_CPU_load)
 {
 	OMRPORT_ACCESS_FROM_OMRPORT(portTestEnv->getPortLibrary());
 
-	/* As per the API specification the first two calls to this API will return a negative portable error code. However
+	/* As per the API specification the first call to this API will return a negative portable error code. However
 	 * for the purposes of this test we will not be testing this. This is because the test infrastructure is setup such
-	 * that we cannot guarantee that no other test has called omrsysinfo_get_CPU_utlization or omrsysinfo_get_CPU_load
-	 * up to this point. If some other test did call these APIs then the internal buffers would have been populated and
-	 * as such the omrsysinfo_get_CPU_load could return a zero return code on the very first invocation within this
-	 * test.
+	 * that we cannot guarantee that no other test has called omrsysinfo_get_CPU_load up to this point. If some other
+	 * test did call these APIs then the internal buffers would have been populated and as such the omrsysinfo_get_CPU_load
```

Comment:
it is singular now since omrsysinfo_get_CPU_utlization has been removed from the previous line
```suggestion
	 * test calls this API then the internal buffers would have been populated and as such the omrsysinfo_get_CPU_load
```